routes/events/index.js

```diff
@@ -98,11 +98,33 @@
       const name = (event.name ? event.name.fi : "") || "";
       const imageUrl = event.images && event.images.length ? event.images[0].url : null;
       const start = formatDate(event.start_time, event.end_time);
+      const place = event.location ? event.location : {};
+      let price = undefined;
+
+      if (event.offers.length > 0) {
```

Comment:
price may sometimes be undefined, is this by design or mistake?

Reply:
There is highly possible that price is undefined because existing data might not be proper enough caused by this bug. Now in pug code there is conditional rendering if price is undefined and it will not be shown in that case.